results/proto/api.proto

```diff
@@ -40,12 +42,14 @@ message GetTaskRunRequest {
 
 message ListTaskRunsRequest {
 	// TODO: document query format.
-	string query = 1;
-	// TODO: page_token, etc.
+	string filter = 1;
+	int32 page_size = 2;
+	string page_token = 3;
```

Comment:
Let's not include any pagination fields in the API until we support this.

Reply:
done